packages/server/test/integration/cypress_spec.js

```diff
@@ -20,7 +20,6 @@ const argsUtil = require(`../../lib/util/args`)
 const { fs } = require(`../../lib/util/fs`)
 const ciProvider = require(`../../lib/util/ci_provider`)
```

Comment:
I notice `  options.browser = options.browser || 'electron'` got lost in `run-ct.ts`

Is this now un-needed?

Reply:
It's the default here: https://github.com/cypress-io/cypress/pull/23002/files#diff-34201b19e236c2f1e4512cce84d30f85425eaa6f293a0b57025f038b59190bf7R20

Reply:
^ link doesn't seem to go to a specific line but I will take your word for it :+1: approved.